cob_hardware_config/cob4-2/config/docking_linear_nav.yaml

```diff
@@ -16,7 +16,7 @@ goal_threshold_rot: 0.08 #0.08
 speed_threshold_rot: 0.01 #0.08
 
 #tf parameters
-global_frame: station_reflector_link #/map
+global_frame: odom_combined #/map
```

Comment:
do you really want to change this? `odom_combined` is not stationary wrt. `map`. Thus, in some edge cases (high slippage, ...) this will actually lead to a very bad performance...